### DISCUSSION TITLE
Contribution reviewers

### DISCUSSION THREAD

**Discussion by AstroTlaloc** [hub4j/github-api#1723] [General]
Hello folks,

I reported an issue regarding the verification status for signed commits. 
Also, created a PR for the fix :)

First time contributing to this project. Do i need to tag anybody as a reviewer? Or send it to a mailing list?

Issue: https://github.com/hub4j/github-api/issues/1714
PR: https://github.com/hub4j/github-api/pull/1715

**Comment by bitwiseman** [comment]
I've been on vacation.  I'll take a look this week.

**Reply by AstroTlaloc** [reply]
Got it. No rush. I was wanted to make sure i got the process right.
Thanks

**Reply by snowden-fu** [reply]
recently I am also trying contributing something but don't know where to get started, hope there is a doc section for How to Contribute helping people.

**Reply by AstroTlaloc** [reply]
@fuzheng1998 i would start here: https://github.com/hub4j/github-api/blob/main/CONTRIBUTING.md
Picking what to work is up to you, you can start looking at the reported issues and see if any of them peak your interest
